tests/docker-build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+# Print executed commands
+set -x
+
+pushd ..
+git clone https://github.com/martinpitt/umockdev.git
```

Comment:
The package available in Debian isn't good enough?
https://packages.debian.org/sid/devel/umockdev

Reply:
Already replied, it does not support thunderbolt yet.

Reply:
It would be good to include a comment about what version or commits are needed. So when they make it into Sid (and promote to testing) this can be changed again.